yarn.lock

```diff
@@ -10252,7 +10252,7 @@ svg-tags@^1.0.0:
   resolved "https://registry.yarnpkg.com/svg-tags/-/svg-tags-1.0.0.tgz#58f71cee3bd519b59d4b2a843b6c7de64ac04764"
   integrity sha1-WPcc7jvVGbWdSyqEO2x95krAR2Q=
 
-svgo@^1.0.0:
+svgo@^1.0.0, svgo@^1.3.2:
```

Comment:
Since we don't need two versions of `svgo` (consumed as a devDependency in `postcss-svgo`) let's try running:

```
[odyssey]$ yarn upgrade svgo@^1.3.2
```

This _should_ bump up the version `postcss-svgo` is looking for, resulting in this lock file change:

```diff
-svgo@^1.0.0, svgo@^1.3.2:
+svgo@^1.3.2:
```

Reply:
Run as above, result of this command was an identical `yarn.lock`, but moved `svgo@^1.3.2` from devDep to dep in `odyssey-icons` + adding `lerna` as a devDep for that package.